package.json

```diff
@@ -15,5 +15,9 @@
   ],
   "author": "",
   "license": "ISC",
-  "dependencies": {}
+  "dependencies": {
+    "react-semantic-toasts": "^0.6.4",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.88.1"
```

Comment:
All of this can be removed, this package.json doesn't need to have any dependencies.

Reply:
it has been added automatically when I install toast , I will remove it.